idea/src/org/jetbrains/kotlin/idea/intentions/AddAccessorIntentions.kt

```diff
@@ -92,15 +58,8 @@ abstract class AbstractAddAccessorsIntention(
     }
 }
 
-private fun createFamilyName(addGetter: Boolean, addSetter: Boolean): () -> String = when {
-    addGetter && addSetter -> KotlinBundle.lazyMessage("text.add.getter.and.setter")
-    addGetter -> KotlinBundle.lazyMessage("text.add.getter")
-    addSetter -> KotlinBundle.lazyMessage("text.add.setter")
-    else -> throw AssertionError("At least one from (addGetter, addSetter) should be true")
-}
-
-class AddPropertyAccessorsIntention : AbstractAddAccessorsIntention(true, true), LowPriorityAction
+class AddPropertyAccessorsIntention : AddAccessorsIntention(true, true), LowPriorityAction
```

Comment:
So in FIR we don't have this `LowPriorityAction`. Any idea if this is important for the "add getter and setter" case?

Reply:
Thanks for noticing this! From the comment it looks like this affects the ordering of entries in the UI. Let me do the same here.